src/claim/Claim.ts

```diff
@@ -18,7 +18,7 @@ import { verifyClaimStructure } from '../ctype/CTypeUtils'
 import Identity from '../identity/Identity'
 import IClaim from '../types/Claim'
 
-function verifyClaim(claimContents: object, cType: CType): boolean {
+function verifyClaim(claimContents: IClaim['contents'], cType: CType): boolean {
```

Comment:
I changed the naming of the argument as it wasn't a claim but the Contents. Which allowed me to add the IClaim contents.